metaspace2020/metaspace
# Added copy dataset/project id button

### Description

In order to improve advanced user experience, an option to copy the dataset_id and project_id values to clipboard should be added, so it could copied to scripts easily. #1047 


#### How to test
1 - Access the dataset page  (<namespace>/datasets) -> click the copy id button (right besides the dataset name on the dataset card) -> the dataset id should be copied to the clipboard
2 - Access the project page  (<namespace>/projects) -> click the copy id button (right besides the project name on the project card) -> the project id should be copied to the clipboard


#### Changes

##### Webapp

###### CopyButton
- [x] Added props to show copy id icon

###### DatasetItemMetadata
- [x] Added copy dataset id button

###### Project list item
- [x] Added copy project id button

- [x] Added multichannel to goblal state


#### Tests
 
##### Front end
 
 
###### Browsers and resolutions

<img width="1184" alt="image" src="https://user-images.githubusercontent.com/35172605/165504506-fffdc4e4-218e-47e9-a61f-ceddb19f60ac.png">
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/35172605/165504623-7fbb42ad-1767-48a9-b104-f3ad63ddded0.png">
